.appveyor.yml

```diff
@@ -1,20 +1,39 @@
-image:
-- Visual Studio 2019
-platform:
-- x64
-configuration:
-- Release
-cache:
-- C:\Tools\vcpkg
+skip_commits:
+  files:
+    - '*travis*'
+    - '*.md'
+    - '.github/**'
+    - '.clang-format'
+    - 'CHANGES'
+    - 'LICENSE'
+
+skip_branch_with_pr: true
+
+image: Visual Studio 2019
+
+environment:
+  QTDIR: C:\Qt\5.15\msvc2019_64
+
+cache: C:\Tools\vcpkg
```

Comment:
Why remove the newline? What if I decide to add things to this later?

Reply:
Reduces line count. Using multi-line syntax for one line is unnecessary.

Reply:
Line count is not exactly a goal here. The "what if I decide to add things to this later" trumps "well maybe we can save a line"